anago

```diff
@@ -200,7 +200,7 @@ check_prerequisites () {
     logecho -n "Checking cloud account/auth $user: "
     if [[ "$gcloud_auth_list" =~ -\ $user ]] && \
        (logrun gcloud config set account $user && \
-        logrun gcloud docker >/dev/null 2>&1); then
+        logrun gcloud docker version >/dev/null 2>&1); then
```

Comment:
```console
$ $ gcloud docker version
WARNING: The '--' argument must be specified between gcloud specific args on the left and DOCKER_ARGS on the right. IMPORTANT: previously, commands allowed the omission of the --, and unparsed arguments were treated as implementation args. This usage is being deprecated and will be removed in March 2017.
This will be strictly enforced in March 2017. Use 'gcloud beta docker' to see new behavior.
```